### ISSUE TITLE
corpse entrance oddity

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1877]
its really annoying when you leave a holy card in the mausoleum / gehenna alt path's mom room and then after it turns into alt path when killing heart everything on the ground just disappears - at least make it 50/50 between disappearing or no

**Comment by Turtier** [comment]
Why would it be 50/50 for disappearing? I'm not against 50/50s for some things like item effects where applicable (crooked penny is a great example), but generally they should be avoided as they would cause unnecessary frustration whenever u miss the 50/50. In this case it would be a small but nice QOL change to keep pickups when the floor changes to corpse after killing heart, but as of rn its just something to keep in mind.

**Comment by Lonrite** [comment]
> Why would it be 50/50 for disappearing? I'm not against 50/50s for some things like item effects where applicable (crooked penny is a great example), but generally they should be avoided as they would cause unnecessary frustration whenever u miss the 50/50. In this case it would be a small but nice QOL change to keep pickups when the floor changes to corpse after killing heart, but as of rn its just something to keep in mind.

yeah I agree p much, i said 50/50 in case they really want the dissappearance thing to cling on
i'm also against it
